evap/grades/templates/grades_semester_view.html

```diff
@@ -48,8 +48,8 @@ <h3 class="panel-title">{% trans "Courses" %}</h3>
                                     <div class="btn-group">
                                         <button type="button" class="btn btn-sm btn-default dropdown-toggle" data-toggle="dropdown" aria-expanded="false" {{ disable_if_archived }}>{% trans "Upload grades" %} <span class="caret"></span></button>
                                         <ul class="dropdown-menu" role="menu">
-                                            <li><a href="{% url "grades:upload_grades" semester.id course.id %}">Midterm grades</a></li>
-                                            <li><a href="{% url "grades:upload_grades" semester.id course.id %}?final=true">Final grades</a></li>
+                                            <li><a href="{% url "grades:upload_grades" semester.id course.id %}">{% trans "Midterm grades" %}</a></li>
+                                            <li><a href="{% url "grades:upload_grades" semester.id course.id %}?final=true">{% trans "Final grades" %}</a></li>
```

Comment:
i absolutely did not check for missing i18n during review, sorry for that. i'll try to remember next time